docs/ops/sort/MulticlassNonMaxSuppression_9.md

```diff
@@ -0,0 +1,208 @@
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_classes, num_boxes]` with box scores. The tensor type should be same with `boxes`. **Required.**
+
+*   **3**: `roisnum` - tensor of type *T_IND* and shape `[num_batches]` with box numbers of each image, which sum up to `num_boxes`. **Required.**
+
+**Outputs**:
+
+*   **1**: `selected_outputs` - tensor of type *T* which should be same with `boxes` and shape `[number of selected boxes, 6]` containing the selected boxes with score and class as tuples `[class_id, box_score, xmin, ymin, xmax, ymax]`.
+
+*   **2**: `selected_indices` - tensor of type *T_IND* and shape `[number of selected boxes, 1]` the selected indices in the flattened `boxes`, which are absolute values cross batches. Therefore possible valid values are in the range `[0, num_batches * num_boxes - 1]`.
+
+*   **3**: `selected_num` - 1D tensor of type *T_IND* and shape `[num_batches]` representing the number of selected boxes for each batch element.
+
+When there is no box selected, `selected_num` is filled with `0`. `selected_outputs` is an empty tensor of shape `[0, 6]`, and `selected_indices` is an empty tensor of shape `[0, 1]`.
+
+**Types**
+
+* *T*: floating-point type.
+
+* *T_IND*: `int64` or `int32`.
+
+**Example**
+
+```xml
+<layer ... type="MulticlassNonMaxSuppression" ... >
+    <data sort_result="score" output_type="i64" sort_result_across_batch="false" iou_threshold="0.2" score_threshold="0.5" nms_top_k="-1" keep_top_k="-1" background_class="-1" normalized="false" nms_eta="0.0"/>
+    <input>
+        <port id="0">
+            <dim>3</dim>
+            <dim>100</dim>
+            <dim>4</dim>
+        </port>
+        <port id="1">
+            <dim>3</dim>
+            <dim>5</dim>
+            <dim>100</dim>
+        </port>
+    </input>
+    <output>
+        <port id="5" precision="FP32">
+            <dim>-1</dim> <!-- "-1" means a undefined dimension calculated during the model inference -->
+            <dim>6</dim>
+        </port>
+        <port id="6" precision="I64">
+            <dim>-1</dim>
+            <dim>1</dim>
+        </port>
+        <port id="7" precision="I64">
+            <dim>3</dim>
+        </port>
+    </output>
+</layer>
+```
+Another possible example with 3 inputs could be like:
+```xml
+<layer ... type="MulticlassNonMaxSuppression" ... >
+    <data sort_result="score" output_type="i64" sort_result_across_batch="false" iou_threshold="0.2" score_threshold="0.5" nms_top_k="-1" keep_top_k="-1" background_class="-1" normalized="false" nms_eta="0.0"/>
+    <input>
+        <port id="0">
+            <dim>3</dim>
+            <dim>100</dim>
+            <dim>4</dim>
+        </port>
+        <port id="1">
+            <dim>3</dim>
+            <dim>100</dim>
+        </port>
+        <port id="2">
+            <dim>10</dim>
```

Comment:
Shouldn't it be 100 or is it dependent on batch number later on? It is quite unclear to me with description(*) of: 
> shape `[num_batches]` with box numbers of each image, which sum up to `num_boxes`.

Reply:
port 2 is num_batches. It is a vector of size num_batches. The elements in it are number of boxes for each image. 

10 here means there are 10 images in this batch. The number of boxes in each image is from the input (varaible).